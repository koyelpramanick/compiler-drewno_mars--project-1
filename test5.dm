""
"&!88"  5 7
    /
"use \n to denote a newline character"
"use \" to  for a quote and \\ for a backslash"
_test
214748364799
@
