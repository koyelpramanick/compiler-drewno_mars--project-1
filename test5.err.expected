FATAL [7,1]-[7,13]: Integer literal overflow
FATAL [8,1]-[8,2]: Illegal character @
